### PR TITLE
AgglomClust error with new sklearn version

### PR DESCRIPTION
**Pull request by microbemarsh**
During a scikit-learn update the affinity argument has been removed and now metric is used instead.

### CODE REVIEW

**Comment by microbemarsh** [comment]
Fixed within PR #11
